### ISSUE TITLE
Multiple Border Types not supported in Table Cell 

### ISSUE THREAD

**Issue by jsvishal** [gitbrent/PptxGenJS#775]
Border:  [ {type:"none", color:'FF0000',pt:1}, {type:"dash", color:'FF0000',pt:1}, {type:"none",color:'0000ff',pt:5}, {type:"dash",color:'9e9e9e',pt:1} ];
		
**Multiple border styles not working.**

![image](https://user-images.githubusercontent.com/7326080/85207778-d6bdc680-b348-11ea-93d7-b0b2a625544b.png)

**Comment by gitbrent** [comment]
Good catch, i did not have a test case for multi-border that used `type:none`

**Comment by gitbrent** [comment]
Fixed in master/v3.3.0

```javascript
var pptx = new PptxGenJS();
var slide = pptx.addSlide();

slide.addTable(
  [
    [
      {
        text: "cell",
        options: {
          border: [
            { type: "none", color: "FF0000", pt: 1 },
            { type: "dash", color: "FF0000", pt: 1 },
            { type: "none", color: "0000ff", pt: 5 },
            { type: "dash", color: "9e9e9e", pt: 1 },
          ],
        },
      },
    ],
  ],
  { x: 1, y: 1, w: "80%", h: 3, align: "center", fill: pptx.SchemeColor.background2 }
);

pptx.writeFile("PptxGenJS-Sandbox-" + getTimestamp()).then(function (fileName) {
  console.log("Saved! File Name: " + fileName);
});
```

<img width="1198" alt="Screen Shot 2020-06-23 at 23 24 43" src="https://user-images.githubusercontent.com/7218970/85500714-af196900-b5a9-11ea-8dd6-46d855d78a17.png">
